pkg/ddl/000038_oidc_user.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE auth_users
+    ADD COLUMN IF NOT EXISTS external_id VARCHAR(255) UNIQUE NULL;
```

Comment:
I don't think we use the `NULL` (non-)constraint in our DDL.  I had to look up what it means, so I slightly prefer that you _remove_ it from here.

Reply:
done